### ISSUE TITLE
ZONE for firewalld in network state

### ISSUE THREAD

**Issue by berkut3128** [salt-formulas/salt-formula-linux#190]
Hi, Can you please add in linux/network/interface.sls support of ZONE for firewalld?
I think code should be like this:
`  {%- if interface.name_zone is defined %}
  - zone: {{ interface.name_zone }}
  {%- endif %}
`
now it does not works:
![2019-04-18 13-32-53](https://user-images.githubusercontent.com/27014426/59901787-fc964f80-9404-11e9-8b9f-5c8c7e1c8ab5.jpg)

**Comment by 0x6c** [comment]
I don't believe it's related directly to linux formula, as this one seems to be a firewalld related.
Conflicting file editions could be a problem, so you will have to control your network interface state from 2 places(firewalld and linux formulas), which is pretty bad.

Anyway, due to simple implementation, I will get this done.
